### PR TITLE
test fixes for 7.21.2

### PR DESCRIPTION
**Pull request by lesterchoi-okta**
## Description:

test fixes for 7.21.2 release

## PR Checklist

- [ ] Have you verified the basic functionality for this change?
- [ ] Did you add tests, as appropriate, following our [Automated Test guidelines](https://oktawiki.atlassian.net/wiki/spaces/eng/pages/2676497890/Automated+Testing+in+the+Signin+Widget)?
- [ ] Did you follow our [Security Best Practices](https://oktawiki.atlassian.net/wiki/display/eng/Security+Best+practices)?
- [ ] Did you verify the change by running [downstream monolith artifact](https://oktawiki.atlassian.net/wiki/spaces/eng/pages/102897979/Sign-in+Widget+Development#Sign-inWidgetDevelopment-Instructionstocreateandrunthedownstreamartifact(d16t))? (Provide link to build below)
- [ ] Does this PR include noticeable changes to the UI? (If yes, attach screenshots/video below)

### Issue:

- [OKTA-796074](https://oktainc.atlassian.net/browse/OKTA-796074)

### CODE REVIEW

File: test/testcafe/spec/DeviceChallengePollView_spec.js [hunk 1]
```diff
@@ -517,6 +517,7 @@ test
       })).eql(1);
   });
 
+// FIXME quarantined test OKTA-796308
 test
```
Comment (shawyu-okta) [inline test/testcafe/spec/DeviceChallengePollView_spec.js:521]: Not a big deal but this test isn't skipped anymore

**Review by shawyu-okta — APPROVED** [review]

File: test/testcafe/spec/DeviceChallengePollView_spec.js (hunk 1 above)
Comment (lesterchoi-okta) [reply test/testcafe/spec/DeviceChallengePollView_spec.js:521]: saw the `.skip` annotation

```
...loopbackPollTimeoutMock).skip('new poll...
```

Comment (shawyu-okta) [reply test/testcafe/spec/DeviceChallengePollView_spec.js:521]: 👓 Gotta get my eyes checked

Comment (lesterchoi-okta) [reply test/testcafe/spec/DeviceChallengePollView_spec.js]: Nope, placement was weird. Just verified locally that `.skip` (where it was) still causes test to be ignored

**Review by shawyu-okta — APPROVED** [review]

**Review by shawyu-okta — APPROVED** [review]

**Pull request merged**